spec/ciFullConf.js

```diff
@@ -31,7 +31,7 @@ exports.config = {
     'tunnel-identifier': process.env.TRAVIS_JOB_NUMBER,
     'build': process.env.TRAVIS_BUILD_NUMBER,
     'name': 'Protractor suite tests',
-    'version': '47',
```

Comment:
This is still a work in progress. Saucelabs doesn't have 51 yet, and the beta appears to be broken now. I'll check back tomorrow.